### ISSUE TITLE
Using dashboardUrl does not function properly

### ISSUE THREAD

**Issue by audiovention** [G4brym/R2-Explorer#46]
I followed the instructions and deployed packages/dashboard in my cloudflare pages.
The initial page loads fine, but navigating is impossible as the routing doesn't function properly when dashboardUrl is specified, 
i.e. requests like mysub.mydomain.com/storage/bucket-name return 404

**Comment by G4brym** [comment]
Can you share the url for your instance in read only

**Comment by audiovention** [comment]
No, sorry. Already moved it behind zero trust. The steps to reproduce are absolutely minimal.
- npm create r2-explorer@latest and deploy
- fork and create Cloudflare page for packages/dashboard (pointing to the master branch - maybe that's a problem, should I use the latest release tag?)
- change dashboardUrl to the cloudflare page url and re-deploy worker

**Comment by audiovention** [comment]
Oh and btw, I've temporarily removed the dashboardUrl option (so I'm using the default public frontend) and everything is working fine.

**Comment by audiovention** [comment]
Figured out my issue, when I deployed to cloudflare pages I didn't have a build step, so I was just uploading the Vue source.
Maybe consider adjusting the docs about that, it's not obvious for non-web people.
